IBM/ibm-aix/tests/CRON_Start.json

```diff
@@ -32,7 +32,7 @@
       "user": {
         "name": "root"
       },
-      "command_line": "cron /usr/share/centrifydc/bin/logrotate.sh 2>&1 >> /var/log/centrify_logrotate.log"
+      "command_line": "cron None /usr/share/centrifydc/bin/logrotate.sh 2>&1 >> /var/log/centrify_logrotate.log"
```

Comment:
We have a problem. `None` doesn't exist in the original message.
I think that
```
%{NOTSPACE:process_name}(\s*%{NOTSPACE:process_file_name})?%{DATA:process_command_line}
```
should become a custom pattern and use it name (with a variable) in the `PROC_Execute` pattern.